client/js/map.js

```diff
@@ -378,6 +378,20 @@ function getClusterRadius(zoom) {
     }
 }
 
+function shortenLargeNumber(number, digits) {
+    var units = ['K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'],
```

Comment:
I don't think we foresee having any trees above the `M` level :)

Reply:
I go save this message if one day pass of M level :D